logical/framework/field_data_test.go

```diff
@@ -246,6 +246,17 @@ func TestFieldDataGet(t *testing.T) {
 			[]string{"abc"},
 		},
 
+		"string slice type, empty string": {
+			map[string]*FieldSchema{
+				"foo": &FieldSchema{Type: TypeStringSlice},
+			},
+			map[string]interface{}{
+				"foo": "",
+			},
+			"foo",
+			[]string{},
+		},
```

Comment:
Could you add a similar test for TypeCommaStringSlice?

Reply:
Done.